frontend_build/src/apiSpecExportTools.js

```diff
@@ -0,0 +1,81 @@
+var fs = require("fs");
+var path = require("path");
+
+/**
+ * The following code is designed to read our apiSpec Javascript, but without having to resolve the
+ * requires contained therein, which include references to files that are not amenable to a vanilla
+ * node js require - however, they are properly handled by our webpack build process.
+ */
+
+// Find the API specification file relative to this file.
+var specFilePath = path.resolve(path.join(__dirname, '../../kolibri/core/assets/src/core-app/apiSpec.js'))
+
+// Read the spec file and do a regex replace to change all instances of 'require('...')'
+// to just be the string of the require path.
+var apiSpecFile = fs.readFileSync(specFilePath, 'utf-8').replace(/require\(('\S+')\)/g, '$1');
```

Comment:
Will our linting rules prevent extra spaces or double quotes from fouling this up?